app/src/main/java/fr/gaulupeau/apps/Poche/service/tasks/GenericFieldsTask.java

```diff
@@ -52,7 +53,7 @@ protected void readFromParcel(Parcel in) {
         genericStringField1 = readString(in);
         genericStringField2 = readString(in);
 
-        genericBooleanField1 = readBoolean(in);
+        genericBooleanField1 = ParcelCompat.readBoolean(in);
```

Comment:
Now it's not compatible with the write method. Revert this one too.

Reply:
Oh. Shame :(